### DISCUSSION TITLE
【询问贴】关于看到绳网APP开发声明后的一个疑问

### DISCUSSION THREAD

**Discussion by dolokong** [share121/inter-knot#2750] [其他]
<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" >
首先感谢创始人大佬开发绳网APP，祝开发顺利，也祝绳网社区越来越好！
其次有一个疑问或者说是请求，看样子绳网APP是准备开发全平台的，整体开发周期应该不会太短，就是说HarmonyOS NEXT版本的app有没有希望呢<img src="https://img-static.mihoyo.com/communityweb/upload/efc2b5f2804b63cbaf377d5cacd5eff6.png" alt="问号" width="75" >，虽然说NEXT还没有正式商用，并且未来的用户数量在短期不会超过安卓和IOS，但应该也会有可观的绳匠用户。如果创始人大佬还有余力的话，可不可以考虑一下再开发一版HarmonyOS NEXT版本的APP呢？不知道现在绳网上有多少 NEXT的潜在用户，刚才在APP开发通知的评论区里有人提过这个。

**Comment by zhixuan45** [comment]
我记得 next 自带一个类墙屏蔽，估计可能连数据总站都访问不了，还是洗洗睡吧...
阿里郎next名不虚传（

**Reply by HKLHaoBin** [reply]
可能会有人去做镜像站，能绕开这层限制

**Reply by zhixuan45** [reply]
> 可能会有人去做镜像站，能绕开这层限制

要钱要资源...还是悬，况且你服务器设置国内基本要备案，不像国外基本不用即开即用，估计还得写屏蔽词什么的

**Reply by dolokong** [reply]
NEXT平板上的红莲花浏览器没有限制，而且支持安装插件。之前老鸿蒙内测版本限制也特别多也有默认的安全浏览，一到正式版也全给开关了，看看NEXT商用版怎么样吧，不行就换设备<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Reply by zhixuan45** [reply]
> NEXT平板上的红莲花浏览器没有限制，而且支持安装插件。之前老鸿蒙内测版本限制也特别多也有默认的安全浏览，一到正式版也全给开关了，看看NEXT商用版怎么样吧，不行就换设备

我持悲观态度

**Comment by HKLHaoBin** [comment]
可能也是基于写网页的逻辑去写移动端的开发的，只不过篡改🐒的功能要单独写。所以，门槛应该不会降低，除非做镜像站。
当然 这些只是猜测 如果我去做的话 我会这样做
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Reply by zhixuan45** [reply]
镜像站？github的镜像似了多少个了，就算只镜像绳网，那也要各种备案，麻烦，非常麻烦

**Reply by zhixuan45** [reply]
况且我记得阿里郎os不支持侧载，也就是阿里郎系统上的绳网app也要申请备案

**Comment by nostalgiatan** [comment]
121用flutter写全平台。

**Reply by zhixuan45** [reply]
阿里郎app要备案的，死了这条心罢
